components/mpas-ocean/src/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -401,19 +444,32 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThickEdgeMea
                tempTracer = tracers(iTr, k, iCell) + dt*(redi_term1(iTr, k, iCell) + redi_term2(iTr, k, iCell) + &
                                                          redi_term3(iTr, k, iCell))/layerThickness(k, iCell)
 
-               if (tempTracer < minimumVal(iTr)) then
+               if (tempTracer < minimumVal(iTr) .or. &
```

Comment:
same here, you need to add things like `over` and `scal` to the OMP private directives I think